server/apps/main/views.py

```diff
@@ -322,22 +326,43 @@ def moderation_list(request):
     else:
         return redirect("main:overview")
 
-# @vcr.use_cassette('tmp/my_stories.yaml', filter_query_parameters=['access_token'])
+
 def my_stories(request):
     """
     List all stories that are associated with the OpenHumans project page.
     Including those which are not-shareable on the website
     """
     if request.user.is_authenticated:
-        context = {"files": request.user.openhumansmember.list_files()}
+        files = request.user.openhumansmember.list_files()
+        context = {"files": files}
         context = reformat_date_string(context)
-        statuses = get_review_status(context["files"])
-        context = {**context, **statuses}
+
+        # Define the number of items per page
+        items_per_page = 10
```

Comment:
Agree, I guess we'll have pagination on a number of pages (moderation, public experiences, "my stories"). For now it might be easiest to have them all use the same parameter. 

Going forward we might want to change the parameter to different ones for different pages, but even then it would probably be nice to have them set up in the settings!

Reply:
Done, thanks!